x-pack/plugins/observability/server/services/slo/sli_client.ts

```diff
@@ -57,13 +60,22 @@ export class DefaultSLIClient implements SLIClient {
   }
 }
 
-function fromSLOTimeWindowToRange(slo: SLO): { from: string; to: string } {
-  if (!slo.time_window.is_rolling) {
-    throw new NotSupportedError(`Time window: ${slo.time_window.is_rolling}`);
+function fromSLOTimeWindowToEsRange(slo: SLO): { from: string; to: string } {
```

Comment:
💬  Main change here: Supports for calendar aligned time window using the introduced date_range domain service